packages/exhibitor/extra/zkServer.sh

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env bash
```

Comment:
copying and hand-modifying this script isn't very useful. Most of this code we don't maintain and hand-updating the changes and figuring out what the implicit diff from the one bundled with zk 3.4.9 isn't going to be fun. Need to reduce this to the single exec line with arguments in it. Vast majority of this script is never touched.